cli/config/configurator.go

```diff
@@ -162,6 +162,19 @@ func (e invalidServerErr) Render() {
 }
 
 func (c Configurator) populateConfigWithVersionInfo(ctx context.Context, cfg Config) (_ Config, _ error, isOSS bool) {
+	cliVersion := Version
+	if cliVersion == "dev" {
```

Comment:
Helper code to help us to run the agent locally when the cliVersion is `dev`

Reply:
Can we point the agent to a different environment from a dev build too? I want to make sure this will allows us to be flexible enough when debugging

Reply:
Yes! Added an update to allow these config to be configured by env vars.